ragaai_catalyst/tracers/agentic_tracing/tracers/custom_tracer.py

```
@@ -284,8 +290,13 @@ def create_custom_component(self, **kwargs):
             "interactions": interactions
         }
 
-        if self.gt:
-            component["data"]["gt"] = self.gt
+        if name in self.span_attributes_dict:
```

Ruff (0.8.2)

Undefined name `name`

(F821)